package.json

```diff
@@ -66,7 +66,7 @@
     "build:watch": "onchange 'src/**/*.js' -i -- yarn build",
     "lint": "./node_modules/eslint/bin/eslint.js .eslint-bin scripts src test",
     "readme:update-coverage": "yarn test:coverage && node ./scripts/update-coverage.js",
-    "test": "jest",
-    "test:coverage": "jest --coverage --collectCoverageFrom=\"src/**/*.{ts,js}\""
+    "test": "jest --runInBand",
+    "test:coverage": "jest --runInBand --coverage --collectCoverageFrom=\"src/**/*.{ts,js}\""
```

Comment:
Do we also have tests with interdependencies in this repository?
This change should have it's own PR and commit message to make all intents atomic (tests contribution / feature contribution).